src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -1010,4 +1003,33 @@ private String getLabelString(Node n) {
         }
     }
 
+    /**
+     * Fixes JENKINS-47779 caused by JENKINS-47713
+     * Not using SortedSet because of PluginWrapper doesn't implements equals and hashCode.
+     * @return new copy of the PluginManager.getPlugins sorted
+     */
+    private static Iterable<PluginWrapper> getPluginsSorted() {
+        PluginManager pluginManager = Helper.getActiveInstance().getPluginManager();
+        return getPluginsSorted(pluginManager);
+    }
+
+    private static Iterable<PluginWrapper> getPluginsSorted(PluginManager pluginManager) {
+        return listToSortedIterable(pluginManager.getPlugins());
+    }
+
+    private static <T extends Comparable<T>> Iterable<T> listToSortedIterable(List<T> list) {
+        final List<T> sorted = new LinkedList<T>(list);
+        Collections.sort(sorted);
+        return sorted;
+    }
+
+    private static void populatePluginsLists(List<PluginWrapper> activePlugins, List<PluginWrapper> disabledPlugins) {
+        for(PluginWrapper plugin : getPluginsSorted()){
+            if(plugin.isActive()) {
+                activePlugins.add(plugin);
+            } else {
+                disabledPlugins.add(plugin);
```

Comment:
You're slightly changing the behaviour here for the Dockerfile, FWIW, from `isEnabled()` to `isActive()`. That means the plugins list will be the one really active currently, not the one that *will* be next time. 

I suppose this is OK and probably what is desired from a support standpoint but well.